models/events.yml

```diff
@@ -114,3 +114,7 @@ models:
         description: '{{ doc("column_initial_referrer_hash_path") }}'
       - name: initial_referrer_hash_query
         description: '{{ doc("column_initial_referrer_hash_query") }}'
+       - name: event_properties
```

Comment:
Looks like we need to fix this indentation so that the YAML can be parsed correctly.

Reply:
this is so embarrassing, thanks for catching it, will fix asap